doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
+  items having a description that matches the given regular expression.
+  Equivalent to *-Q --filter-pattern* 'regex'.
 
 *--status*::
-  Display  the  status of running instances of calcurse. If calcurse is
-  running, this will tell  if  the  interactive mode  was  launched  or  if
-  calcurse is running in background.  The process pid will also be indicated.
-
-*-t*[num], *--todo*[=num]::
-  Print the *todo* list and exit. If the optional number 'num' is given, then
-  only uncompleted todos having a priority equal to 'num' will be returned. The
-  priority number must be between 1 (highest) and 9 (lowest). It is also
-  possible to specify *0* for the priority, in which case only completed tasks
-  will be shown. Equivalent to *-Q --filter-type todo*, combined with
-  *--filter-priority* and *--filter-completed* or *--filter-uncompleted*.
-
-*--to* <date>::
-  Specify the end date of the range when used with *-Q*. Cannot be combined
-  with *--days*.
+  Display the status of running instances of calcurse, interactive or
+  background mode. The process pid is also printed.
+
+*-t*['num'], *--todo*[='num']::
+  Print the *todo* list. If the optional number 'num' is given, then only
+  uncompleted (open) todos having a priority equal to 'num' will be returned.
+  The priority number must be between +1+ (highest) and +9+ (lowest). It is
+  also possible to specify +0+ for the priority, in which case only completed
+  (closed)  tasks will be shown. Equivalent to *-Q --filter-type todo*,
+  combined with *--filter-priority* and *--filter-completed* or
+  *--filter-uncompleted*.
+
+*--to* 'date'::
+  Specify the end date of the day range when used with *-Q*. When used without
+  *--from* the start day is the current day. Cannot be combined with *--days*,
+  see <<_query,-Q --query>>.
 
 *-v*, *--version*::
-  Display *calcurse* version and exit.
+  Display *calcurse* version.
+
+*-x*['format'], *--export*[='format']::
+  Export user data in the specified format. Events, appointments and todos are
+  converted and echoed to stdout. Two formats are available: +ical+ and
+  +pcal+. The default 'format' is +ical+.
+
+[[_limiting_options]]
+.Filter, format and day range options
+*************************************
+These options do not accomplish anything in and of themselves. They influence
+other options and are in a way opposites: filter options affect the input to,
+format and day range options the output from calcurse. Specifically, filter
+options limit what is loaded from the data files into calcurse, day range
+options limit what is output (see <<_query,-Q>>), and
+<<_format_options,format options>> describe how it is presented.
+
+Filter options have effect on queries (*-Q* and query short-forms), grep
+(*-G*), purge (*-P*) and export (*-x*). Format options have effect on queries,
+grep and *--dump-imported*. Day range options have effect on queries only.
+*************************************
```

Comment:
Again, I do not think this is very useful in practice. Let's rather turn all subsections into proper sections and move the pieces of information from this box to the intros of the corresponding new sections (where not present already).